extensions/2.0/Vendor/EXT_primitive_voxels/README.md

```diff
@@ -65,13 +69,19 @@ The `dimensions` property of the extension specifies the voxel grid dimensions:
 
 Dimensions must be nonzero. Elements are laid out in memory first-axis-contiguous, e.g. for boxes, `x` data is contiguous (up to stride).
 
+The `bounds` property describes how the voxel is clipped before rendering in voxel space. `bounds.minimum` and `bounds.maximum` specify a "rectangular" region of the voxel in the appropriate
+coordinate systems:
+- Boxes: a rectangular region of the voxel, between `(-1, -1, -1)` and `(1, 1, 1)`
```

Comment:
I wonder if boxes should omit this property

Reply:
Starting to change my mind on this. I think `bounds` can be useful for the box shape when the voxel glTFs are coming from a tileset. The base transformation can come from the tileset and the node transforms can set to identity. This will be more consistent with how ellipsoid and cylinder tilesets are set up, where there's no other way to define the shape of individual tiles besides using `bounds`.